lib/index.js

```diff
@@ -388,6 +391,17 @@ MapboxGeocoder.prototype = {
     return this.options.proximity;
   },
 
+  /**
+   * Get the language to use in UI elements and when making search requests
+   * 
+   * Look first at the explicitly set options otherwise use the browser's language settings
+   */
```

Comment:
For completeness given this is a public method, I think we should add JSDoc for `@returns`.